build.sh

```diff
@@ -49,8 +49,6 @@ fi
 echo "::endgroup::"
 
 echo "::group::Adding PPA..."
-echo "Adding PPA: $REPOSITORY"
-add-apt-repository -y ppa:$REPOSITORY
```

Comment:
Is adding the current PPA repository a problem for you?

Reply:
1. We have extra_ppa to do this work. Just add the target PPA to extra_ppa is OK.
2. Add the target ppa is unnecessary for publishing current project to PPA.

Reply:
The most important reason is 3. When you create a new PPA without any deb, then you add-apt-repository it will meet error. because this github action is run in ubuntu focal, however this PPA doesn't have any deb for focal (in fact, there is no any deb).

Reply:
However, once you dput a deb of focal to this PPA, then add-apt-repository will not meet this error.